docker/addons/influxdb-writer/docker-compose.yml

```diff
@@ -23,6 +23,10 @@ services:
       INFLUXDB_ADMIN_PASSWORD: mainflux
     networks:
       - docker_mainflux-base-net
+    expose:
```

Comment:
Same here.

Reply:
I found that a bit weird too in the [mongodb config](https://github.com/mainflux/mainflux/blob/master/docker/addons/mongodb-writer/docker-compose.yml#L20) , but as it was already there, I added the ports to the other configs the same way for consistency. Reading the docker-compose docs again I found out that ports does the same as expose plus it exposes the port to the host. I will fix the mongodb config as well.